src/components/Login/UserDropdown.tsx

```diff
@@ -1,6 +1,6 @@
+import Avatar from "react-avatar";
```

Comment:
Probably it's the package name to use as the sorting key, not the items imported..